platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -921,6 +1270,7 @@ namespace platoon_strategic_ihp
         // Check joining plan type.
         bool isFrontJoin = (plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_FROM_FRONT);
         bool isRearJoin = (plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_AT_REAR);
+        bool isCutinJoin = (plan_type.type == cav_msgs::PlanType::CUT_IN_FROM_SIDE);
 
         // rear join; platoon leader --> leader waiting
         if (isRearJoin)
```

Comment:
There is a TODO on line 1281 that I believe needs to be resolved before this plugin is complete.  Do you agree?

Reply:
This TODO comment is actually carried over from the previous platooning code. The lane information is measured using cross track. I will delete this obsolete comment.